github/AuthenticatedUser.py

```diff
@@ -763,10 +766,14 @@ def get_authorizations(self):
     def get_emails(self):
         """
         :calls: `GET /user/emails <http://developer.github.com/v3/users/emails>`_
-        :rtype: list of string
+        :rtype: list of namedtuples with members email, primary, verified and visibility
         """
         headers, data = self._requester.requestJsonAndCheck("GET", "/user/emails")
-        return data
+        emaildata = []
+        itemdata = namedtuple("EmailData", "email primary verified visibility")
+        for item in data:
+            emaildata.append(itemdata._make(item.values()))
```

Comment:
`return [itemdata._make(item.values()) for item in data]` saves having to populate a second list, we can just return a list comprehension.